src/QbXml/Objects/Types/DATETIMETYPE.cs

```diff
@@ -167,7 +167,9 @@ public void ReadXml(System.Xml.XmlReader reader)
             reader.ReadStartElement();
             if (!isEmptyElement)
             {
-                value = DateTime.Parse(reader.ReadContentAsString());
+                DateTime dt;
+                DateTime.TryParse(reader.ReadContentAsString(), out dt);
+                value = dt;
```

Comment:
I guess this will leave the DateTime as a default struct. So you are not changing any functionality. DATETIMETYPE.ToString() still work right?